Source/Plugins/CrossPlatformPluginGenerator/Packages/packages-lock.json

```diff
@@ -0,0 +1,387 @@
+{
+  "dependencies": {
+    "com.unity.2d.sprite": {
+      "version": "1.0.0",
+      "depth": 0,
+      "source": "builtin",
+      "dependencies": {}
+    },
+    "com.unity.2d.tilemap": {
+      "version": "1.0.0",
+      "depth": 0,
+      "source": "builtin",
+      "dependencies": {}
+    },
+    "com.unity.ads": {
```

Comment:
Could probably prune lots of these if you wanted

Reply:
Added #36988141 to track this.